src/styles.css

```diff
@@ -397,11 +397,16 @@ a.dropdown-item:hover {
 
   .plugin-info {
     color: gray;
-    font-size: 0.75rem;
+    font-size: 0.8rem;
   }
 
   .plus-icon {
-    font-size: 0.70rem;
+    font-size: 1rem;
```

Comment:
We could define sizing classes for all bi icons similar to FogLAMP Manage: [https://github.com/dianomic/fogman-ui/pull/538/files#r1333119093](url)